catalog-info.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: "sscs-tribunals-frontend"
```

Is this name accurate?

```suggestion
  name: "sscs-submit-your-appeal"
```

---

it's done based on this:
https://github.com/hmcts/sscs-submit-your-appeal/blob/529640645daaa2c9489f2cbe5a19063361095ed6/Jenkinsfile_CNP#L9-L10

given it's deployed as tribunals-frontend its best to keep this name I think:
https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/sscs/sscs-tribunals-frontend/sscs-tribunals-frontend.yaml